rr_light/run_gui.py

```diff
@@ -0,0 +1,94 @@
+import sys
+
+from PyQt5 import QtWidgets
+
+from rr_light.gui import Ui_MainWindow
+from rr_light.reflection_refraction import (
+    calc_ref_plot,
+    calculate_refr_angl,
+    check_invalid_alfa,
+    check_invalid_index_of_refr,
+    matplotlib_graph,
+)
+
+
+def get_vars(my_app) -> tuple[float, float, float]:
+    """Getting the variables from the user.
+
+    :param self: self
+    :return: Angle of incidence.
+    :return: Index of refraction of the first medium
+    :return: Index of refraction of the second medium
+    """
+
+    alfa = float(my_app.lineEdit.text())
+    check_invalid_alfa(alfa)
+
+    n1 = float(my_app.lineEdit_2.text())
+    n2 = float(my_app.lineEdit_3.text())
+    check_invalid_index_of_refr(n1, n2)
+
+    return alfa, n1, n2
+
+
+class MyApp(Ui_MainWindow):
+    """This is class for the Qt5 app.
+
+    :param Ui_MainWindow: The window that is going to pop up
```

Comment:
In the docsting of a class the parameters correspond to these from the `__init__`; i.e. here it should be `window`. Not sure of the type. You have to check.